.github/workflows/continuous-integration.yml

```diff
@@ -426,45 +426,51 @@ jobs:
 
   # Merge into "trying" branch (bors try) does not trigger release_* jobs.
 
-  bors_try_success:
+  bors_success:
     name: bors build finished
     needs:
-      - run_unit_tests
-      - run_integration_tests
-    if: "github.event_name == 'push' && github.ref == 'refs/heads/trying' && success()"
+      - release_docker_image
```

Comment:
With bors now, there can be only one test named 'bors build finished'. If there are multiple and one or more skips, bors considers it a failure. One way to fix it would be to have 'release docker image' execute as a standalone test for ref == staging and have bors_success just depend on unit and integration tests. That way, if release docker image fails on a merge, bors will fail. Hopefully.

Reply:
I think we can release Docker image on bors Try also.